core/src/main/java/com/sequenceiq/cloudbreak/job/stackpatcher/ExistingStackPatcherJob.java

```diff
@@ -103,18 +106,21 @@ private void unscheduleJob(JobExecutionContext context) {
                 .findFirst();
     }
 
-    private void applyStackPatch(ExistingStackPatchService existingStackPatchService, Stack stack) throws JobExecutionException {
+    private boolean applyStackPatch(ExistingStackPatchService existingStackPatchService, Stack stack) throws JobExecutionException {
         StackPatchType stackPatchType = existingStackPatchService.getStackPatchType();
         if (!existingStackPatchService.isStackAlreadyFixed(stack)) {
             try {
                 if (existingStackPatchService.isAffected(stack)) {
                     LOGGER.debug("Stack {} needs patch for {}", stack.getResourceCrn(), stackPatchType);
                     stackPatchUsageReporterService.reportAffected(stack, stackPatchType);
-                    existingStackPatchService.apply(stack);
-                    LOGGER.info("Stack {} was patched successfully for {}", stack.getResourceCrn(), stackPatchType);
-                    stackPatchUsageReporterService.reportSuccess(stack, stackPatchType);
+                    boolean success = existingStackPatchService.apply(stack);
+                    if (success) {
```

Comment:
Shouldn't we report unsuccessful patches?

Reply:
We should only report failures, see on line 128. Success can be false in cases like where the stack is not in a correct state to even try to run the patch